### PR TITLE
Changes that correct the conic function

### PR DESCRIPTION
**Pull request by pyrito**
This PR makes the appropriate changes to correct the issues with the conic function for NCTCOG related functionality. These changes are independent from the heap changes that need to be made.

### CODE REVIEW

File: src/tap.c [hunk 1]
```diff
@@ -200,29 +200,28 @@ double quarticBPRint(struct arc_type *arc, bool includeFixedCost) {
...
+    } else if (v_s < 0.925) {
+        time += arc->CD + v_s * (arc->CC + v_s * (arc->CB + v_s * arc->CA));
     } else {
         time += arc->sParam / 0.1;
     }
 
     /* Add unsignalized delay */
-        time += arc->m + arc->u * arc->flow/arc->capacity;
-    
+    time += arc->m + arc->u * v_s;
```
Comment (ribsthakkar) [inline src/tap.c]: arc->u is multiplied with x not v_s

**Review by ribsthakkar — COMMENTED** [review]
Also update convexcombinations.c to use bisection instead of newtonSearch.
Also update intialize CCParameters to use BellmanFord instead of Dijkstra

File: src/tap.c (hunk 1 above)
Comment (pyrito) [reply src/tap.c]: That's a good catch. Thanks

**Review by ribsthakkar — COMMENTED** [review]
Change FW and BFW to bisection as well. After that we can merge

**Review by ribsthakkar — APPROVED** [review]

**Pull request merged**